apps/storefront/src/components/cart/ShoppingCartSummary.tsx

```diff
@@ -171,33 +128,32 @@ const CartSummary: React.FC<CartSummaryProps> = ({
       <Stack spacing={3}>
         <Flex justify="space-between">
           <Text>Subtotal</Text>
-          <Text>${order.Subtotal?.toFixed(2)}</Text>
+          <Text>${orderWorksheet?.Order.Subtotal?.toFixed(2)}</Text>
         </Flex>
-        {order.PromotionDiscount && order.PromotionDiscount > 0 && (
-          <Flex justify="space-between">
-            <Text>Promotion Discount</Text>
-            <Text>- ${order.PromotionDiscount?.toFixed(2)}</Text>
-          </Flex>
-        )}
+        {orderWorksheet?.Order.PromotionDiscount &&
+          orderWorksheet?.Order.PromotionDiscount > 0 && (
+            <Flex justify="space-between">
+              <Text>Promotion Discount</Text>
+              <Text>
+                - ${orderWorksheet?.Order.PromotionDiscount?.toFixed(2)}
+              </Text>
+            </Flex>
+          )}
         <Flex justify="space-between">
           <Text>Shipping</Text>
           {tabIndex !== TABS.SHIPPING ||
             (tabIndex !== TABS.INFORMATION && <Text></Text>)}
-          <Text>
-            {order.ShippingCost === 0
-              ? "FREE SHIPPING"
-              : "$" + order.ShippingCost}
-          </Text>
+          <Text>${orderWorksheet?.Order.ShippingCost.toFixed(2)}</Text>
```

Comment:
ShippingCost and TaxCost are both nullable so let's add optional chaining to be safe